src/renderer/features/wallets/WalletSelect/ui/WalletGroup.tsx

```diff
@@ -54,8 +54,9 @@ export const WalletGroup = ({ type, wallets }: Props) => {
                   )
                 }
                 onClick={() => walletSelectModel.events.walletSelected(wallet.id)}
-                onInfoClick={() => walletSelectModel.events.walletIdSet(wallet.id)}
```

Comment:
Now we can pass whatever button is required as children and not rely on tricky behavior of `WalletCardMd`